### PR TITLE
added favicon

### PR DESCRIPTION
**Pull request by niteeshpatil**
added favicon,this is my first time,tell me any corrections are required

### CODE REVIEW

**Comment by Subhradeep10** [comment]
@niteeshpatil Share the screenshot/ result of your following changes

**Comment by Subhradeep10** [comment]
@niteeshpatil And always ask at the issue first, then we will assign it to you. Cause here we do first come first serve

**Comment by Subhradeep10** [comment]
@niteeshpatil  Anyway your work is good, But that are some gentle advice for you, cause it is your first time in open source

File: Button Counter/index.html [hunk 1]
```diff
@@ -5,6 +5,7 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="shortcut icon" type="image/png" href="../img/Favicon.png">
```
Comment (Subhradeep10) [inline Button Counter/index.html:8]: `<link rel="shortcut icon" type="image/png" href="../img/Favicon.png">`

Add 
`<link rel="icon" type="image" href="../img/Favicon.png">`

**Comment by niteeshpatil** [comment]
Screenshot (383)
Screenshot (384)

I changed code as u suggested & attached result/screenshot

**Comment by Subhradeep10** [comment]
@niteeshpatil Great 🚀🚀

**Pull request merged**